library/std/math.qs

```diff
@@ -2,8 +2,49 @@
 // Licensed under the MIT License.
 
 namespace Microsoft.Quantum.Math {
+    open Microsoft.Quantum.Convert;
     open Microsoft.Quantum.Diagnostics;
 
+    //
+    // Constants PI, E, LogOf2.
+    //
+
+    /// # Summary
+    /// Represents the ratio of the circumference of a circle to its diameter.
+    ///
+    /// # Ouptut
+    /// A double-precision approximation of the the circumference of a circle
+    /// to its diameter, π ≈ 3.14159265358979323846.
+    ///
+    /// # See Also
+    /// - Microsoft.Quantum.Math.E
+    function PI() : Double {
+        3.14159265358979323846
+    }
+
+    /// # Summary
+    /// Returns the natural logarithmic base to double-precision.
+    ///
+    /// # Output
+    /// A double-precision approximation of the natural logarithic base,
+    /// e ≈ 2.7182818284590452354.
+    ///
+    /// # See Also
+    /// - Microsoft.Quantum.Math.PI
+    function E() : Double {
+        2.7182818284590452354
+    }
+
+    /// # Summary
+    /// Returns the natural logarithm of 2.
+    ///
+    /// # Output
+    /// Returns a `Double` equal to 0.6931471805599453.
+    function LogOf2 () : Double
+    {
+        return 0.6931471805599453;
```

Comment:
```suggestion
        0.6931471805599453
```

Reply:
nit: since in general in the standard library we are now using implicit returns.

Reply:
This applies to other functions as well.

Reply:
Removed all unnecessary return statements.